src/main/java/org/broadinstitute/hellbender/engine/filters/MappingQualityReadFilter.java

```diff
@@ -20,10 +21,10 @@
 public final class MappingQualityReadFilter extends ReadFilter implements Serializable {
     private static final long serialVersionUID = 1L;
 
-    @Argument(fullName="minimum-mapping-quality", doc = "Minimum mapping quality to keep (inclusive)", optional=true)
+    @Argument(fullName=ReadFilterArgumentDefinitions.MAXIMUM_MAPPING_QUALITY_NAME, doc = "Minimum mapping quality to keep (inclusive)", optional=true)
     public int minMappingQualityScore = 10;
 
-    @Argument(fullName="maximum-mapping-quality", doc = "Maximum mapping quality to keep (inclusive)", optional=true)
+    @Argument(fullName=ReadFilterArgumentDefinitions.MINIMUM_MAPPING_QUALITY_NAME, doc = "Maximum mapping quality to keep (inclusive)", optional=true)
```

Comment:
BUGBUG: you've swapped the minimum and maximum mapping quality arguments here!

Reply:
Since this wasn't caught by any tests, could you please add a good unit test for `MappingQualityReadFilter` that fails on the current version of this branch, and passes if you label the maximum and minimum args correctly.

Reply:
Ups! There is a test that is failing due to this change (after fixing compilation), but it is not testing explicitly the `--maximum-mapping-quality` argument but the `--minimum-mapping-quality`. Is that enough test for this?

I can fill in an issue to add another test for this param and add it after this PR. I really need this change in for the next point release.

Reply:
Since the next point release is not until Monday, you should have time to add an additional test in this PR.